### PR TITLE
Feat/app tile UI

### PR DESCRIPTION
**Pull request by Ldoppea**
This PR add UI improvement on Tiles as they are displayed in `cozy-store`:

- Highlight app tiles that have available update (`update` text is now displayed in primaryColor)
image
- Highlight app tiles that are installed (tile background is now grey)
image
- Allow app tiles's names to be displayed on two line when no `status` (installed or update) is set
image

Changes can be observed here:

https://ldoppea.github.io/cozy-ui/react/#!/AppTile

https://ldoppea.github.io/cozy-ui/react/#!/AppSections

### CODE REVIEW

**Comment by Ldoppea** [comment]
Notables approved changes on Argos:
- `TileTitle` is now 1px taller (height) than before as we don't fix its height to 1.25rem (to allow him to grow to 2 lines). This produce a lot of small diffs:
image
- We can notice that one of the tiles' icon is smaller than other on mobile view. This is because I created a sample with very large title and this show an old bug unrelated to this PR. I expect to fix the bug this afternoon on another PR
image

File: react/AppTile/index.jsx [hunk 1]
```diff
@@ -67,6 +67,8 @@ export const AppTile = ({
       className={
         isInMaintenanceWithSpecificDisplay
           ? styles['AppTile-container-maintenance']
+          : statusLabel === 'installed'
+          ? styles['AppTile-container-installed']
           : undefined
```
[the remark this answers is not in the document]
Comment (Ldoppea) [reply react/AppTile/index.jsx]: Bien vu, c'est corrigé (bon par contre j'aime pas le formatage prettier)

File: react/Tile/styles.styl [hunk 2]
```diff
@@ -70,7 +70,6 @@ $tileMobileHeight = 3.75rem
   max-width 100%
 
 .Tile-title
-  height 1.25rem
   color var(--black)
```
[the remark this answers is not in the document]
Comment (Merkur39) [reply react/Tile/styles.styl:73]: Ah pardon oui effectivement, c'était dans l'idée d'amélioration 😇

[the remark this answers is not in the document]
Comment (Ldoppea) [reply react/Tile/styles.styl:73]: Ok, ping @JF-Cozy des fois que tu soit déjà passé dessus dans ta tache d'implémentation des nouvelles palettes?

**Review by Merkur39 — APPROVED** [review]

File: react/Tile/styles.styl (hunk 2 above)
[the remark this answers is not in the document]
Comment (JF-Cozy) [reply react/Tile/styles.styl:73]: Celui-ci n'a pas bougé non, faut plutôt voir avec @joel-costa s'il est censé être noir (ce qui n'est pas impossible) ou plutôt changeant en fonction du thème

**Pull request merged**